README.md

```diff
@@ -48,13 +48,165 @@ MobileCoin Full Service is available under open-source licenses. Look for the [L
+
+```sh
+curl -s localhost:9090/wallet \
+  -d '{
+        "method": "set_password",
+        "params": {
+          "password": "TestPassword&"
+        }
+      }' \
+  -X POST -H 'Content-type: application/json' | jq
+
+{
+  "method": "set_password",
+  "result": {
+    "success": true
+  }
+}
+```
+
+| Optional Param  | Purpose                  | Requirements              |
+| :-------------  | :----------------------- | :------------------------ |
+| `password`      | Password for the encrypted materials in the database.   |  |
+
+#### Unlock
+
+If your wallet was locked previously via `set_password`, then you will need to unlock it before you can interact with your accounts.
+
+```sh
+curl -s localhost:9090/wallet \
+  -d '{
+        "method": "unlock",
+        "params": {
+          "password": "TestPassword&"
+        }
+      }' \
+  -X POST -H 'Content-type: application/json' | jq
+
+{
+  "method": "unlock",
+  "result": {
+    "success": true
+  }
+}
+```
+
+| Optional Param  | Purpose                  | Requirements              |
+| :-------------  | :----------------------- | :------------------------ |
+| `password`      | Password for the encrypted materials in the database.   | |
+
+##### Troubleshooting
+
+If you receive the following error, this means you have never set up a password for your wallet. You will need to [`set_password`](#set-password), which will also unlock the database.
+
+```sh
+{
+  "error": "Database(SetPassword)"
+}
+```
+
+If you receive the following error, it means you provided the wrong `password`:
+
+```sh
+{
+  "error": "Database(PasswordFailed)",
+  "details": "Error interacting with the database: Password failed"
+}
+```
+
+If you receive the following error, it means you are attempting an action that requires the database to be unlocked:
+
+```sh
+{
+  "error": "Database(NoDecryptionKey)",
+  "details": "Error interacting with the database: No decryption key in database."
+}
+```
+
+#### Change Password
+
+```sh
+curl -s localhost:9090/wallet \
+  -d '{
+        "method": "change_password",
+        "params": {
+          "old_password": "TestPassword&",
+          "new_password": "RotatedPassword@"
+        }
+      }' \
+  -X POST -H 'Content-type: application/json' | jq
+
+{
+  "method": "change_password",
+  "result": {
+    "success": true
+  }
+}
+```
+
+| Optional Param  | Purpose                  | Requirements              |
+| :-------------  | :----------------------- | :------------------------ |
+| `old_password`      | Password for the encrypted materials in the database.   | |
+| `new_password`      | Password for the encrypted materials in the database.   | |
+
+##### Troubleshooting
+
+If you receive the below, it means you provided the wrong `old_password`:
+
+```sh
+{
+  "error": "Database(PasswordFailed)",
+  "details": "Error interacting with the database: Password failed"
+}
+```
+
+#### Is Locked
+
+Check whether the database is currently locked.
+
+```sh
```

Comment:
This is not working for me:
```
curl -s localhost:9090/wallet \
  -d '{
        "method": "is_locked",
      }' \
  -X POST -H 'Content-type: application/json'

            <!DOCTYPE html>
            <html lang="en">
            <head>
                <meta charset="utf-8">
                <title>422 Unprocessable Entity</title>
            </head>
            <body align="center">
                <div role="main" align="center">
                    <h1>422: Unprocessable Entity</h1>
                    <p>The request was well-formed but was unable to
                be followed due to semantic errors.</p>
                    <hr />
                </div>
                <div role="contentinfo" align="center">
                    <small>Rocket</small>
                </div>
            </body>
            </html>
```

Reply:
```
2021-01-21 19:09:05.452479 UTC ERRO Couldn't parse JSON body: Error("unknown variant `is_locked`, expected one of `set_password`, `unlock`, `change_password`, `get_locked_status`, `create_account`, `import_account`, `get_all_accounts`, `get_account`, `update_account_name`, `delete_account`, `get_all_txos_by_account`, `get_txo`, `get_wallet_status`, `get_balance`, `create_address`, `get_all_addresses_by_account`, `send_transaction`, `build_transaction`, `submit_transaction`, `get_all_transactions_by_account`, `get_transaction`, `get_transaction_object`, `get_txo_object`, `get_block_object`, `get_proofs`, `verify_proof`", line: 2, column: 29), mc.app: full-service, mc.module: rocket_contrib::json, mc.src: /Users/eran/.cargo/registry/src/github.com-1ecc6299db9ec823/rocket_contrib-0.4.6/src/json.rs:153
```

Reply:
Ah thank you - need to update the documentation. It's `get_locked_status` now